hugo/layouts/index.html

```diff
@@ -47,21 +46,6 @@ <h2 class="pl-2 pb-2">Welcome to the ACL Anthology!</h2>
       {{ .Content }}
     </aside>
     -->
-
-    <div class="row mb-3">
-      {{ if (fileExists "/data-export/anthology.bib.gz") }}
-      <div class="col-6 col-xl-12 mb-2">
-        <a class="btn btn-block btn-info" href="{{ "anthology.bib.gz" | relURL }}">Full Anthology as BibTeX ({{ printf "%.2f MB" (div (os.Stat "/data-export/anthology.bib.gz").Size 1000000.0) }})</a>
-        <a class="btn btn-block btn-info" href="{{ "anthology+abstracts.bib.gz" | relURL }}">…with abstracts ({{ printf "%.2f MB" (div (os.Stat "/data-export/anthology+abstracts.bib.gz").Size 1000000.0) }})</a>
-      </div>
-      {{ end }}
```

Comment:
What is the rationale behind removing these entirely from the front page?

Reply:
Just aesthetics, really. I could restore it if you like, but I would probably change them up.

Reply:
Do you think people ever download those by clicking on the links? What if we replaced the button link target to one that points to the "citation files" section of the FAQ?

Reply:
I would assume so, but the server logs should know best ;)

Maybe a small italics note “Looking for bib files? See this FAQ” or something could also work, if you don’t want the buttons.